test_haystack/whoosh_tests/test_whoosh_backend.py

```diff
@@ -752,6 +757,11 @@ def test_scoring(self):
             ["0.40", "0.40", "0.40"],
         )
 
+    def test_analyzed_fields(self):  # TODO: rename to test_analyzed_fields
```

Comment:
It looks like you already got to this TODO:

```suggestion
    def test_analyzed_fields(self):
```

Reply:
Good catch, will fix.